low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -30,77 +31,131 @@ from framework.base.sspl_constants import PRODUCT_FAMILY, SUPPORT_REQUESTOR_NAME
 # Add the top level directories
 sys.path.insert(0, f'/opt/seagate/{PRODUCT_FAMILY}/sspl/low-level')
 from framework.utils.config_reader import ConfigReader
+onGoingRequestPath = "/tmp/sspl-sb-req.txt"
 
-SYS_INFORMATION = "SYSTEM_INFORMATION"
-IEM_SENSOR = "IEMSENSOR"
+class SSPLBundleGenerate(object):
 
-localTempPath = "/tmp/support_bundle/"
-sspl_log_default = f"/var/log/{PRODUCT_FAMILY}/sspl"
-iem_log_default = f"/var/log/{PRODUCT_FAMILY}/iem"
-onGoingRequestPath = "/tmp/sspl-sb-req.txt"
-sspl_conf_dir = "/opt/seagate/{PRODUCT_FAMILY}/provisioner/pillar/components/sspl.sls"
-rabbitmq_conf_dir = "/etc/rabbitmq/rabbitmq.config"
-rabbitmq_log_dir = "/var/log/rabbitmq"
-rabbitmq_status = "/tmp/rabbitmq_status.txt"
-enclosure_log = "/tmp/enclosure_logs.zip"
-enclosure_wwn = "NA"
-rabbitmq_cmd = {
-    "Rabbitmq status": "systemctl status rabbitmq-server",
-    "Rabbitmq cluster status": "rabbitmqctl cluster_status",
-    "Rabbitmq node status": "rabbitmqctl status",
-    "Rabbitmq environment": "rabbitmqctl environment",
-    "Rabbitmq report": "rabbitmqctl report",
-    "Lists all queues in the current node": "rabbitmqctl list_queues",
-    "Lists all vhosts in the current node": "rabbitmqctl list_vhosts",
-    "Lists all rabbitmq channels in virtual host SSPL": "rabbitmqctl list_channels -p SSPL",
-    "Lists all users": "rabbitmqctl list_users"
-}
-
-def make_sspl_support_bundle_tarfile(parser, tmpFile):
-    file_name = "sspl_%s.tar.gz" % parser[1]
-    try:
-        conf_reader = ConfigReader()
-
-    except (IOError, ConfigReader.Error) as err:
-        print("[ Error ] when validating the configuration file %s :" % \
-            sspl_conf_dir)
-        print(err)
-        print("Exiting ...")
-        sys.exit(1)
+    def __init__(self):
 
-    sspl_log_dir = conf_reader._get_value_with_default(SYS_INFORMATION,
-        "sspl_log_file_path", sspl_log_default).replace("/sspl.log","")
+        self.SYS_INFORMATION = "SYSTEM_INFORMATION"
+        self.IEM_SENSOR = "IEMSENSOR"
+        self.localTempPath = "/tmp/support_bundle/"
+        self.sspl_log_default = f"/var/log/{PRODUCT_FAMILY}/sspl"
+        self.iem_log_default = f"/var/log/{PRODUCT_FAMILY}/iem"
+        self.sspl_conf_dir = f"/opt/seagate/{PRODUCT_FAMILY}/provisioner/pillar/components/sspl.sls"
+        self.rabbitmq_conf_dir = "/etc/rabbitmq/rabbitmq.config"
+        self.rabbitmq_log_dir = "/var/log/rabbitmq"
+        self.rabbitmq_status = "/tmp/rabbitmq_status.txt"
+        self.ipmi_sel_data = f"/tmp/ipmi_sel_data_{str(int(time.time()))}.txt"
+        self.consul_folder_name = 'consul_sspl_data'
+        self.consul_sspl_config = "/tmp/consuldump_sspl_config/"
```

Comment:
make sure we clean these tmp files/folders after bundling

Reply:
Done